pkgs/by-name/ra/rails-new/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  unstableGitUpdater,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "rails-new";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "rails";
+    repo = "rails-new";
+    tag = "v${version}";
+    hash = "sha256-7hEdLu9Koi2K2EFIl530yA+BGZmATFCcBMe3htYb0rs=";
+  };
+
+  useFetchCargoVendor = true;
+
+  cargoHash = "sha256-FrndtE9hjP1WswfOYJM4LW1UsE8S9QXthYO7P3nzs2I=";
+
+  passthru.updateScript = unstableGitUpdater { };
```

```suggestion
  passthru.updateScript = nix-update-script { };
```
this is not an unstable package. Unstable packages are those who do not have release tags